### PR TITLE
Add calls for migrating minichef rewards to new meta pools

### PR DESCRIPTION
**Pull request by penandlim**
Also adds new pools to the migrator contract for easier LP migration

### CODE REVIEW

File: scripts/2022_05_12_migrate_metapool_rewards.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    # outdated tbtc meta pool
+    minichef.set(5, 0, "0x0000000000000000000000000000000000000000", False)
+    # outdated susd meta pool
+    minichef.set(6, 0, "0x0000000000000000000000000000000000000000", False)
+    # outdated wcusd meta pool
+    minichef.set(7, 0, "0x0000000000000000000000000000000000000000", False)
+
+    # Add new rewards for the meta pools
+    # tbtc v3 meta pool lp token
+    minichef.add(25, "0xA2E81Eb93F0F9814ae9A3bea2D2A63408f2709C1",
```
Comment (hammeiam) [inline scripts/2022_05_12_migrate_metapool_rewards.py:29]: ✅

File: scripts/2022_05_12_migrate_metapool_rewards.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                 "0x0000000000000000000000000000000000000000")
+    # susd v3 meta pool lp token
+    minichef.add(50, "0x444F94460a641429CDa4e38E02E51642Cc38276A",
```
Comment (hammeiam) [inline scripts/2022_05_12_migrate_metapool_rewards.py:32]: ✅

File: scripts/2022_05_12_migrate_metapool_rewards.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                 "0x0000000000000000000000000000000000000000")
+    # wcusd v3 meta pool lp token
+    minichef.add(2, "0x0dB8b09c13FE21913faF463274cE8e0a51719f16",
```
Comment (hammeiam) [inline scripts/2022_05_12_migrate_metapool_rewards.py:35]: should this be so low?

Comment (hammeiam) [reply scripts/2022_05_12_migrate_metapool_rewards.py:35]: ✅  address

**Review by hammeiam — APPROVED** [review]

File: scripts/2022_05_12_migrate_metapool_rewards.py (hunk 3 above)
Comment (penandlim) [reply scripts/2022_05_12_migrate_metapool_rewards.py:35]: image
Yep allocpoint is set to 2 for wcusd pool

**Review by ug02fast — APPROVED** [review]

**Review by ug02fast — APPROVED** [review]

**Review by hammeiam — APPROVED** [review]
lgtm

**Pull request merged**